doc/nrf/known_issues.rst

```diff
@@ -341,6 +341,22 @@ Immutable bootloader board restrictions
 Build system
 ============
 
+.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
+
+KRKNWK-7827: Application build system is not aware of the settings partition
+  The application build system is not aware of partitions, including the settings partition, which for big applications can result in application code overlapping with other partitions.
+  As a consequence, writing to overlapping partitions might remove or damage parts of the firmware, which can lead to errors that are difficult to debug.
+
+  **Workaround:** Define and use a code partition.
+  You can use one of the following options:
+
+  * :ref:`partition_manager` from |NCS|.
+    For example, for single image, you can use :option:`CONFIG_PM_SINGLE_IMAGE`.
+  * Devicetree code partition from Zephyr.
+    Set :option:`CONFIG_USE_DT_CODE_PARTITION` Kconfig option to ``y``
```

Comment:
I checked if there is anything useful about the code partition in Zephyr's documentation but it seems there is almost nothing.
So I guess it would be good to mention that one has to define a code partition with appropriate proprieties (offset, size) (see legacy_flash_partitions from doc/guides/dts/legacy-macros.rst) and then choose it as code partition using `zephyr,code-partition` on `chosen` node (see 'aliases and chosen nodes' in doc/guides/dts/intro.rst).